webpack.mix.js

```diff
@@ -84,6 +84,16 @@ mix.webpackConfig({
  |
  */
 
-mix.js('resources/assets/js/app.js', 'public/js/app.js')
-   .sass('resources/assets/sass/app.scss', 'public/css/app.css')
-   .sass('resources/assets/sass/home.scss', 'public/css/home.css');
\ No newline at end of file
+mix.setPublicPath('public')
+   .js('resources/assets/js/app.js', 'js')
+   .js('resources/assets/js/event_view.js', 'js')
+   .js('resources/assets/js/filebrowser.js', 'js')
+   .js('resources/assets/js/pilots_guide.js', 'js')
+   .js('resources/assets/js/realops.js', 'js')
+   .js('resources/assets/js/snowstorm.js', 'js')
+   .sass('resources/assets/sass/app.scss', 'css')
+   .sass('resources/assets/sass/clipboard.scss', 'css')
+   .sass('resources/assets/sass/Footer-white.scss', 'css')
+   .sass('resources/assets/sass/home.scss', 'css')
+   .sass('resources/assets/sass/main.scss', 'css')
+   .sass('resources/assets/sass/pilots_guide.scss', 'css');
```

Comment:
Is there a way to get mix to pick up all of these at once? My thought is maybe

```js
mix.setPublicPath('public')
  .js('resources/assets/js/*', 'js')
  .sass('resources/assets/sass/*', 'css');
```

Reply:
@iccowan I just looked at the documentation - wildcards are not allowed in the path. Please let me know if you see something different.
https://laravel-mix.com/docs/main/examples

Reply:
Hmm I wonder if we can leverage `mix.combine()` here to effectively get the result we want. I don't see it in the official docs, but I see it here as an example (so no promise that it actually exists). But in `mix.combine()`, wildcards are supposed to work

https://laracasts.com/discuss/channels/elixir/mix-all-js-in-directory

```js

mix.js('resources/assets/js/app.js', 'public/js')
   .sass('resources/assets/sass/app.scss', 'public/css');
mix.combine([
    'public/css/bootstrap.min.css', 
    'public/css/style.css',
    'public/css/animate.css'
], 'public/css/app.css');
mix.combine([
    'public/js/jquery-2.1.1.js',
    'public/js/bootstrap.min.js',
    'public/js/plugins/metisMenu/jquery.metisMenu.js',
    'public/js/plugins/slimscroll/jquery.slimscroll.min.js',    
    'public/js/plugins/pace/pace.min.js'
], 'public/js/app.js');
```

Reply:
@iccowan I found a work-around. We still want to keep the app.js and app.css separate - they contain global scripts. For the individual includes, I put them in a `mix` directory.
We don't want to use the `mix.combine()` - from what I read that will aggregate the scripts together and I haven't QC'd them to ensure that they will work as one.

Reply:
Sweet! This looks good. I'll QA this and get it approved